### PR TITLE
Update to 1.35.0

### PR DESCRIPTION
**Pull request by ndoschek**
- Update Theia to 1.35.0
- Update playwright to 1.30.0 and and open up version range
- Update allure versions

Signed-off-by: Nina Doschek <ndoschek@eclipsesource.com>

### CODE REVIEW

**Comment by ndoschek** [comment]
Hi @planger, @jfaltermeier!
Could you please review this change? It updates the Theia Playwright Template to Theia 1.35.0 and the playwright/test and allure versions. Thanks!

**Comment by planger** [comment]
:+1:  Looks good to me, thanks! I'm not a Theia committer, so I can't approve this change. But checked the code and tested against current Theia master, and found no issues.

**Review by jfaltermeier — APPROVED** [review]

**Comment by ndoschek** [comment]
Hi @jfaltermeier, could you please merge, as I am not a commiter either. Thank you!

**Pull request merged**
